pages/_en-US/ds-index/hardmod.md

```diff
@@ -11,4 +11,91 @@ Hardmodding is when you physically solder the motherboard of the Nintendo DSi to
 ![Original DS motherboard with touch-points labeled](https://media.discordapp.net/attachments/547986366357700620/736370094392999947/mobo_pinout.png)
 
 ### Nintendo DSi
-Follow the "Hardware NAND Mod Guide" and "Decrypting your NAND" of [this](https://gbatemp.net/threads/dsi-downgrading-the-complete-guide.393682/) guide.
+
+- To hardmod a Nintendo DSi you will need:
+  - A DSi or DSi XL (LL)
+  - Pencil tipped soldering iron
+  - Very fine wire, 28AWG or smaller, preferably 30AWG+
+  - SD to microSD adapter
+  - An SD card reader able to read eMMC chips running in single data-line mode.
+  - [Win32DiskImager](https://sourceforge.net/projects/win32diskimager/) and [HxD](https://mh-nexus.de/en/downloads.php?product=HxD20) if on Windows
+  - A valid DSi backup of the DSi you are hardmodding
+  - You must be comfortable soldering to pads that are as small as 0.5mm in diameter
+
+#### DSi side A pinouts
+![DSi side A pinouts](https://safan41.github.io/hardmodguide/images/pinouts/DSi/a.jpg)
+#### DSi side B pinouts
+![DSi side B pinouts](https://safan41.github.io/hardmodguide/images/pinouts/DSi/b.png)
+#### DSi XL side B pinouts
+![DSi XL side B pinouts](https://safan41.github.io/hardmodguide/images/pinouts/DSi%20XL/nand.png)
```

Comment:
Probably better to reupload these here under `/assets/images/` instead of just linking some other site that could change or be deleted or so btw